src/lambda_handler.py

```diff
@@ -27,12 +27,21 @@
 import json
 import logging
 import os
+<<<<<<< HEAD
+=======
+from typing import Any, Optional, Union
+
+# Third-Party Libraries
+import cowsay
+import cowsay.characters
+>>>>>>> e8fed7e947c8a5d19ce75381e0137f1e4fbb83a8
```

Comment:
@jmorrowomni - just flagging that there are several merge conflicts in this PR that must be resolved.

Reply:
@dav3r - The merge conflicts should all be fixed now.